terragraph_planner/los/helper.py

```diff
@@ -546,17 +547,19 @@ def compute_los_batch(
     if use_ellipsoidal_los_model:
         los_validator = EllipsoidalLOSValidator(
             surface_elevation,
-            max_los_distance,
-            min_los_distance,
+            max_los_distance * (1 + DISTANCE_TOLERANCE_PERCENT),
+            min_los_distance * (1 - DISTANCE_TOLERANCE_PERCENT),
+            ELE_SCAN_ANGLE_LIMIT + ELE_SCAN_ANGLE_TOLERANCE,
```

Comment:
Since it's always the internal constant, I think we can remove it from the arg list of the constructor and directly use the constant when checking. But I see the advantage here is more flexibility, so it's up to you to decide where to put it.

Reply:
I think I'll expose this as a user parameter in a subsequent PR.